tests/app/questionnaire/test_routing_path.py

```diff
@@ -1,42 +1,47 @@
 from unittest import TestCase
 from app.questionnaire.routing_path import RoutingPath
-from app.questionnaire.location import Location
 
 
 class TestRouter(TestCase):
     def setUp(self):
-        self.locations = [
-            Location(section_id="section-1", block_id="block-a"),
-            Location(section_id="section-1", block_id="block-b"),
-            Location(section_id="section-1", block_id="block-c"),
-            Location(section_id="section-1", block_id="block-b"),
-            Location(section_id="section-1", block_id="block-c"),
-        ]
-        self.routing_path = RoutingPath(self.locations)
+        self.block_ids = ["block-a", "block-b", "block-c", "block-b", "block-c"]
+        self.section_id = "section-1"
+        self.list_item_id = "list_item_id"
+        self.list_name = "list_name"
+        self.routing_path = RoutingPath(
+            self.block_ids,
+            section_id=self.section_id,
+            list_item_id=self.list_item_id,
+            list_name=self.list_name,
+        )
         super().setUp()
 
     def test_eq_to_routing_path(self):
         self.assertEqual(self.routing_path, self.routing_path)
 
     def test_eq_to_list(self):
-        self.assertEqual(self.locations, self.routing_path)
+        self.assertEqual(self.block_ids, self.routing_path)
+
+    def test_eq_to_tuple(self):
+        self.assertEqual(tuple(self.block_ids), self.routing_path)
 
     def test_len(self):
-        self.assertEqual(len(self.locations), len(self.routing_path))
+        self.assertEqual(len(self.block_ids), len(self.routing_path))
 
     def test_reversed(self):
         self.assertEqual(
-            list(reversed(self.locations)), list(reversed(self.routing_path))
+            list(reversed(self.block_ids)), list(reversed(self.routing_path))
         )
 
     def test_contains_true(self):
-        self.assertIn(self.locations[0], self.routing_path)
-        self.assertNotIn(
-            Location(section_id="section-1", block_id="block-z"), self.routing_path
-        )
+        self.assertIn(self.block_ids[0], self.routing_path)
+        self.assertNotIn("block-z", self.routing_path)
 
     def test_iter(self):
-        self.assertEqual(self.locations[0], next(iter(self.routing_path)))
+        self.assertEqual(self.block_ids[0], next(iter(self.routing_path)))
 
     def test_getitem(self):
-        self.assertEqual(self.locations[0], self.routing_path[0])
+        self.assertEqual(self.block_ids[0], self.routing_path[0])
+        self.assertEqual(self.section_id, self.routing_path.section_id)
+        self.assertEqual(self.list_item_id, self.routing_path.list_item_id)
+        self.assertEqual(self.list_name, self.routing_path.list_name)
```

Comment:
These aren't testing the getitem method and should be a separate test e.g. test_properties.